docs/getting-started/index.md

```diff
@@ -3,52 +3,56 @@ title: Getting started
 section: getting-started
 ---
 
-Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Currently, Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
+Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
 
-## Install
+## Requirements
 
-Shopify App CLI installs using a shell script. Download and run it in your terminal with one command:
+- [Ruby](https://www.ruby-lang.org) 2.5.1+ 
+- [Node.js](https://nodejs.org) 10.0.0+
+- [curl](https://curl.haxx.se) 7.0.0+
```

Comment:
@tanema I threw this in because it's used in the official install process, and I picked the version number out of thin air just because it felt weird not to include one. But as far as I can tell you could be running a curl version from 20 years ago and it would still work since the use case is so constrained. Should we change the version, drop the version, drop curl altogether from this list?

Reply:
Can you just drop the version? Any version of curl would work. For now we do need curl, hopefully we can remove it in the future.